lib/experimental/Information/Counter/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { cva } from "class-variance-authority"
+import React from "react"
+
+interface CounterProps {
+  size?: "md" | "sm"
+  type?: "default" | "selected" | "bold"
+  value: number
+  maxValue?: number
+}
```

Comment:
Please use `VariantProps`:

```suggestion
type CounterProps = {
  value: number
  maxValue?: number
} & VariantProps<typeof counterVariants>
```

Reply:
Aaah that's super handy, didn't know about this! Thanks :D